core/utils.py

```diff
@@ -160,36 +160,42 @@ def create_embed(
         *,
         error: bool = False,
         author: Union[discord.Member, discord.User] = None,
-        **kwargs,
+        title: str = discord.Embed.Empty,
+        description: str = discord.Embed.Empty,
```

Comment:
The type hints are wrong here. You are specifying just a `str`, but default is embed empty. They should be ...`: Union[str, discord.Embed.Empty] = `...